edwin/client/static/js/components/Timeline.js

```diff
@@ -82,6 +99,22 @@ Timeline.propTypes = {
   params: React.PropTypes.object.isRequired,
 };
 
+class Indicator extends React.Component {
+  render() {
+    let state = this.props.state.toJS().state;
```

Comment:
I did it this way to make it easier to add other kinds of state in the future.

Having said that, the "state" value is a Constant and I had a hell of a time trying to get those to do the right thing in regards to comparison and rendering.